### ISSUE TITLE
Donations to OpenCollective Dropdown Hide

### ISSUE THREAD

**Issue by SudharakaP** [opencollective/opencollective#3886]
**Describe the bug**

When we donate to a collective there's a drop-down to contribute a percentage to Open Collective. Also the amount in the drop down is reflected on the right hand side. 

![path839](https://user-images.githubusercontent.com/12435965/105403351-7db75900-5bdd-11eb-8fb1-6323273c30cd.png)

This doesn't make much sense when we want to donate to Open Collective as the above screenshot shows.

**Expected behavior**

Ideally I think the form should hide the sections above when we donate to Open Collective itself. 

**To Reproduce**
1. Go to https://opencollective.com/opencollective/donate
2. Notice the drop down and and Contribution Summary sections.

**Comment by Betree** [comment]
Good point, platform tips when contributing to Open Collective don't make much sense 😄

**Comment by saideepesh000** [comment]
hey @Betree @SudharakaP,  can I try this

**Comment by Betree** [comment]
> hey @Betree @SudharakaP, can I try this

Sure!

**Comment by SudharakaP** [comment]
> hey @Betree @SudharakaP, can I try this

Thanks for your help @saideepesh000 😄

**Comment by saideepesh000** [comment]
Hey @SudharakaP @Betree, it's already solved in the repo code, checkout the pic

Repo master branch code:

![Screenshot (7)](https://user-images.githubusercontent.com/43727167/106143214-76252080-6198-11eb-89e9-4d07c9a794a4.png)

**Comment by snitin315** [comment]
@saideepesh000 feel free to create a PR.

**Comment by snitin315** [comment]
Yep seems to be fixed on staging too - https://staging.opencollective.com/opencollective/donate

**Comment by Betree** [comment]
It's not fixed in prod: https://opencollective.com/opencollective/donate. We should check what's the difference between both envs.

**Comment by snitin315** [comment]
It seems its hidden for every collective on staging - https://staging.opencollective.com/webpack/donate
